### ISSUE TITLE
Navier Stokes data explanation

### ISSUE THREAD

**Issue by kallegrens** [neuraloperator/neuraloperator#123]
Hello!

We have a hard time trying to understand the different features of the Navier Stokes data sets found [here](https://drive.google.com/drive/folders/1UnbQh2WWc6knEHbLn-ZaXrKUZhp7pjt-)  on your Google Drive.

We have a .mat-file which includes:

- `t` (shape 1x20) - simply the time steps in seconds?
- `a` (shape 1200x64x64), 1200 separate initial conditions for the vorticity z-component over the x-y-plane of size 64x64?
- `u` (shape 1200x64x64x20), 1200 different time series of the vorticity z-component over the same computation domain?

In other words, does `a` and `u` contain the same physical quantity (vorticity z-component)?

**Comment by zongyi-li** [comment]
Yes it is correct!
